addons/realisticnames/stringtable.xml

```diff
@@ -2162,147 +2162,147 @@
         </Key>
         <Key ID="STR_ACE_RealisticNames_arifle_SPAR_01_blk">
             <English>HK416A5 10" (Black)</English>
-            <German>HK416A5 10" (Černá)</German>
-            <Spanish>HK416A5 10" (Noir)</Spanish>
-            <Polish>HK416A5 10" (Negro)</Polish>
-            <Czech>HK416A5 10" (Чёрный)</Czech>
-            <French>HK416A5 10" (Czarny)</French>
-            <Russian>HK416A5 10" (Schwarz)</Russian>
-            <Portuguese>HK416A5 10" (Nero)</Portuguese>
+            <Czech>HK416A5 10" (Černá)</Czech>
+            <French>HK416A5 10" (Noir)</French>
+            <Spanish>HK416A5 10" (Negro)</Spanish>
+            <Russian>HK416A5 10" (Чёрный)</Russian>
+            <Polish>HK416A5 10" (Czarny)</Polish>
+            <German>HK416A5 10" (Schwarz)</German>
+            <Italian>HK416A5 10" (Nero)</Italian>
             <Hungarian>HK416A5 10" (Fekete)</Hungarian>
-            <Italian>HK416A5 10" (Preto)</Italian>
+            <Portuguese>HK416A5 10" (Preto)</Portuguese>
         </Key>
         <Key ID="STR_ACE_RealisticNames_arifle_SPAR_01_khk">
             <English>HK416A5 10" (Khaki)</English>
+            <Czech>HK416A5 10" (Khaki)</Czech>
+            <French>HK416A5 10" (Kaki)</French>
+            <Spanish>HK416A5 10" (Caqui)</Spanish>
+            <Russian>HK416A5 10" (Хаки)</Russian>
+            <Polish>HK416A5 10" (Khaki)</Polish>
             <German>HK416A5 10" (Khaki)</German>
-            <Spanish>HK416A5 10" (Kaki)</Spanish>
-            <Polish>HK416A5 10" (Caqui)</Polish>
-            <Czech>HK416A5 10" (Хаки)</Czech>
-            <French>HK416A5 10" (Khaki)</French>
-            <Russian>HK416A5 10" (Khaki)</Russian>
-            <Portuguese>HK416A5 10" (Khaki)</Portuguese>
+            <Italian>HK416A5 10" (Khaki)</Italian>
             <Hungarian>HK416A5 10" (Khaki)</Hungarian>
-            <Italian>HK416A5 10" (Caqui)</Italian>
+            <Portuguese>HK416A5 10" (Caqui)</Portuguese>
         </Key>
         <Key ID="STR_ACE_RealisticNames_arifle_SPAR_01_snd">
             <English>HK416A5 10" (Sand)</English>
-            <German>HK416A5 10" (Písková)</German>
-            <Spanish>HK416A5 10" (Beige)</Spanish>
-            <Polish>HK416A5 10" (Arena)</Polish>
-            <Czech>HK416A5 10" (Песочный)</Czech>
-            <French>HK416A5 10" (Sand)</French>
-            <Russian>HK416A5 10" (piaskowy)</Russian>
-            <Portuguese>HK416A5 10" (Sabbia)</Portuguese>
+            <Czech>HK416A5 10" (Písková)</Czech>
+            <French>HK416A5 10" (Beige)</French>
+            <Spanish>HK416A5 10" (Arena)</Spanish>
+            <Russian>HK416A5 10" (Песочный)</Russian>
+            <Polish>HK416A5 10" (Sand)</Polish>
+            <German>HK416A5 10" (piaskowy)</German>
```

Comment:
This is polish. I think `Sand` is the German one. `Beige` would be valid too, so I'm not sure.